libs/platform/auth/login/login.component.ts

```diff
@@ -87,76 +101,73 @@ export class AuthLoginComponent extends ContentMixin<LoginOptions>(LitElement) {
       )}
 
       <form @submit=${this.onSubmit}>
-        <oryx-input
-          .label=${this.i18n('user.login.email')}
-          required
-          ?hasError="${this.hasError}"
-        >
-          <input
-            type="email"
-            name="email"
-            required
-            placeholder=${this.i18n('user.login.email')}
-          />
-        </oryx-input>
-
-        <oryx-password-input
-          .strategy="${this.$options()?.passwordVisibility}"
-          .label=${this.i18n('login.password')}
-          required
-          ?hasError="${this.hasError}"
-        >
-          <input
-            type="password"
-            name="password"
-            required
-            placeholder=${this.i18n('login.password')}
-          />
-        </oryx-password-input>
-
-        ${this.renderLoginOptions()}
+        ${this.fieldRenderer.buildForm(this.getFields())}
 
         <oryx-button .size=${ButtonSize.Md} ?loading=${this.isLoading}>
           <button slot="custom" ?disabled=${this.isLoading}>
             ${this.i18n('user.log-in')}
           </button>
         </oryx-button>
-      </form>`;
-  }
 
-  protected renderLoginOptions(): TemplateResult | void {
-    if (
-      !this.$options()?.forgotPasswordLink &&
-      !this.$options()?.enableRememberMe
-    ) {
-      return;
-    }
-    return html`
-      <div class="options">
-        ${when(
-          this.$options()?.enableRememberMe,
-          () => html`<oryx-checkbox>
-            <input
-              type="checkbox"
-              name="rememberme"
-              aria-label=${this.i18n('user.login.remember-me')}
-              ${this.rememberme}
-            />
-            ${this.i18n('user.login.remember-me')}
-          </oryx-checkbox>`
-        )}
         ${when(
           this.$options()?.forgotPasswordLink,
-          () => html`<oryx-link>
-            <a href=${this.$options()?.forgotPasswordLink}
-              >${this.i18n('user.login.forgot-password?')}</a
-            >
-          </oryx-link>`
+          () => html`
+            <oryx-button href=${this.$options()?.forgotPasswordLink} type=${
+            ButtonType.Text
+          }>
+              ${this.i18n('user.login.forgot-password?')}</a
+            </oryx-button>
+          `
         )}
-      </div>
+      </form>
+
+      <oryx-heading>
+        <h2>${this.i18n('user.login.new-customer')}</h2>
+      </oryx-heading>
+      <p>${this.i18n('user.login.benefits-of-registering-to-the-website')}</p>
+
+      <oryx-button
+        href=${this.$options()?.registrationLink}
+        type=${ButtonType.Outline}
+      >
+        ${this.i18n('user.login.create-account')}
+      </oryx-button>
```

Comment:
We should not add this content to the login component. Instead, we like to have the following:
- use the `oryx-content-text` component for the title, text and button
- add the component to the login page, only add it for >= 1.1 version

This makes the content content manageable, as well as the layout. Some stores use the text next to the login, in a 50-50 layout.